mixer/pkg/attribute/bag_test.go

```diff
@@ -37,8 +36,8 @@ var (
 func TestBag(t *testing.T) {
 	sm1 := mixerpb.StringMap{Entries: map[int32]int32{-16: -16}}
 	sm2 := mixerpb.StringMap{Entries: map[int32]int32{-17: -17}}
-	m1 := map[string]string{"N16": "N16"}
-	m3 := map[string]string{"N42": "FourtyTwo"}
+	m1 := StringMap{entries: map[string]string{"N16": "N16"}}
```

Comment:
Worth using the WrapStringMap function here and in other similar places?

Reply:
Done